libraries/protocol/asset_ops.cpp

```diff
@@ -211,6 +211,9 @@ void bitasset_options::validate() const
    FC_ASSERT(force_settlement_offset_percent <= GRAPHENE_100_PERCENT);
    FC_ASSERT(maximum_force_settlement_volume <= GRAPHENE_100_PERCENT);
 
+   if( extensions.value.margin_call_fee_ratio.valid() )
+      FC_ASSERT( *extensions.value.margin_call_fee_ratio <= GRAPHENE_COLLATERAL_RATIO_DENOM );
```

Comment:
This check is too strict. Imagining an asset with `MCR=4.0` and `MSSR=2.1`, `MCFR` can be as large as `1.1`.
IMO we can just check like other ratios, requires it to be between `GRAPHENE_MIN_COLLATERAL_RATIO` and `GRAPHENE_MAX_COLLATERAL_RATIO`.

Reply:
Sorry, I was wrong. MCFR can be less than `GRAPHENE_MIN_COLLATERAL_RATIO`.